tests/Controller/ProductVariantApiTest.php

```diff
@@ -423,7 +423,7 @@ public function it_allows_updating_information_about_product_variant()
 <<<EOT
         {
             "code": "NEW_MUG_CODE",
-            "version": $version
+            "version": "$version"
```

Comment:
Why is that necessary? It isn't API version.

Reply:
its failing without it for some reason :/